neo/Network/NeoNode.py

```diff
@@ -256,19 +258,18 @@ def MessageReceived(self, m):
         elif m.Command == 'getheaders':
             self.HandleGetHeadersMessageReceived(m.Payload)
         elif m.Command == 'headers':
-            headerResultDeferred = threads.deferToThread(self.HandleBlockHeadersReceived, m.Payload)
-            headerResultDeferred.addErrback(self.onThreadDeferredErr)
+            reactor.callFromThread(self.HandleBlockHeadersReceived, m.Payload)
```

Comment:
I think this might have to be `callInThread` because `self.HandleBlockHeadersReceived` is not an already running thread.

`callFromThread` is described as:

> From another thread, execute a function in the reactor thread.
> Use callFromThread to call reactor APIs from outside the reactor thread. For ex‐
> ample, use callFromThread when:
> • writing data out through a transport from another thread
> • invoking a custom log observer from another thread
> • stopping the reactor from another thread

whereas `callInThread`says:

> Execute a blocking function in its own thread
which is what I believe we're after.